ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -941,3 +942,84 @@ func convertOnlineRoundParamsTail(ctx context.Context, e db.Executable) error {
 	_, err := e.ExecContext(ctx, createVoteLastValidIndex)
 	return err
 }
+
+func accountsAddCreatableTypeColumn(ctx context.Context, e db.Executable, populateColumn bool) error {
+	// Run ctype resources migration if it hasn't run yet
+	var creatableTypeOnResourcesRun bool
+	err := e.QueryRow("SELECT 1 FROM pragma_table_info('resources') WHERE name='ctype'").Scan(&creatableTypeOnResourcesRun)
+	if err == nil {
+		// Already exists.
```

Comment:
Have a query that checks whether there are any -1 entries for ctype, fail with useful error (corruption, what happened, maybe state remediation statement?).

Reply:
Addressed in latest push